frontend/src/common/utils/nameFormat.ts

```diff
@@ -0,0 +1,10 @@
+export const isNameValid = (name: string) : boolean => {
+    name = name.toLowerCase();
+    for (let i = 0; i < name.length; ++i) {
+        const c = name.charAt(i);
+        if (c.toLowerCase() === c.toUpperCase()) {
+            return false;
+        }
+    }
+    return true;

```

Comment:
could you explain a bit about how this works or add some brief function docs?

Reply:
That's just checking if a character is A-Z or a-z

Reply:
hmm. is there a cleaner way to write this?

Reply:
Not sure, but I'll add a comment explaining the logic.